snuba/state/cache/abstract.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABC, abstractmethod
```

Comment:
For some reason, I thought that git didn't actually make any distinction between `mv` and an `add`/`rm` of a file with identical contents. I might be wrong about this, though (I did not check.) If that's the case, it may have just been the other edits to this file subsequently threw off the similarity algorithm when looking at the patch as a whole.